clang_tools/main.py

```diff
@@ -66,13 +68,19 @@ def main():
     if args.uninstall:
         uninstall_clang_tools(args.uninstall, args.directory)
     elif args.install:
-        install_clang_tools(
-            args.install,
-            args.tool,
-            args.directory,
-            args.overwrite,
-            args.no_progress_bar,
-        )
+        if parse_version(args.install) != (0, 0, 0):
```

Comment:
This new helper function (`parse_version()`) is actually called twice (or more) if the version is not a path: Its called once here and called again in `clang_tools.install.is_installed()`.

It might be better to just pass the resulting tuple to `install_clang_tools()` to avoid parsing the same version string multiple times.

Reply:
done.

I also ran mypy on the pkg/tests sources, just to be sure.